<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Baymax</class>
 <widget class="QMainWindow" name="Baymax">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 238, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>10</y>
      <width>371</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>5a8c65c0b15d5c051b368fb1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Zekton&quot;;
border-color: rgb(0, 0, 0);
selection-color: rgb(255, 255, 255);
background-color: rgb(119, 118, 123);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="feverButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Zekton&quot;;</string>
    </property>
    <property name="text">
     <string>Fever</string>
    </property>
   </widget>
   <widget class="QPushButton" name="headacheButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Zekton&quot;;</string>
    </property>
    <property name="text">
     <string>Head-ache</string>
    </property>
   </widget>
   <widget class="QPushButton" name="diarrheaButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Zekton&quot;;</string>
    </property>
    <property name="text">
     <string>Diarrhea</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vomitButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Zekton&quot;;</string>
    </property>
    <property name="text">
     <string>Vomiting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="coldButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Zekton&quot;;</string>
    </property>
    <property name="text">
     <string>Cold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="injuryButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Zekton&quot;;</string>
    </property>
    <property name="text">
     <string>Injuries</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allergyButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Zekton&quot;;</string>
    </property>
    <property name="text">
     <string>Allergies</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unknownButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Zekton&quot;;</string>
    </property>
    <property name="text">
     <string>Unknown</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
